### DISCUSSION TITLE
Crystallographic Orientation of Lamellae

### DISCUSSION THREAD

**Discussion by janzenchoi** [neperfepx/neper#505] [Q&A] [answered]
Hello,

I am aware that the direction of the lamellae can be specified. However, is it possible to specify the crystallographic orientation of lamellae relative to the parent grain?

Thanks.

**Comment by rquey** [comment, marked as answer]
See [-ori](https://neper.info/doc/neper_t.html#cmdoption-ori) for the current possibilities.

**Reply by janzenchoi** [reply]
Hello @rquey,

Thanks for the suggestion. It was very helpful.

I am trying to visualise my volume such that the colour of the grains represent their crystallographic orientation, based on the IPF colour scheme. I am using the following code.

`neper -V output.tess -datacellcol ori -datacellcolscheme 'ipf(y)' -cameraangle 14.5 -imagesize 800:800 -print img`

Does this look like the correct way of doing it? Also, is it possible to do the same colour scheming with meshes?

**Reply by rquey** [reply]
Yes, this is correct. For meshes, use `elset` instead of `cell` in options: `-datacellcol` becomes `-dataelsetcol`, etc.
